src/benchmarks/real-world/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks/Harness/Config.cs

```diff
@@ -0,0 +1,53 @@
+﻿using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using Microsoft.ML.Benchmarks.Harness;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class RecommendedConfig : ManualConfig
+    {
+        public RecommendedConfig()
+        {
+            Add(DefaultConfig.Instance); // this config contains all of the basic settings (exporters, columns etc)
+
+            Add(GetJobDefinition() // job defines how many times given benchmark should be executed
+                .WithCustomBuildConfiguration(GetBuildConfigurationName())
+                .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark
+
+            Add(new ExtraMetricColumn()); // an extra colum that can display additional metric reported by the benchmarks
```

Comment:
this feature is not used in the benchmarks that you have ported, it can be removed
```suggestion
```